frontend/src/app/components/stock/stock.component.ts

```diff
@@ -0,0 +1,124 @@
+import { Component, OnInit } from '@angular/core';
+import { Company } from 'src/app/models/company-model';
+import { Exchange } from 'src/app/models/exchange-model';
+import { Stock } from 'src/app/models/stock-model';
+import { AuthService } from 'src/app/services/auth.service';
+import { CompanyService } from 'src/app/services/company.service';
+import { ExchangeService } from 'src/app/services/exchange.service';
+import {Router} from "@angular/router"
+
+@Component({
+  selector: 'app-stock',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405405)
```suggestion
  selector: "app-stock",
```